index.html

```diff
@@ -1,19 +1,106 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <main>
+        <header>
+            <div class="main1">
+
+                <img src="img/karma-logo.svg" class="img_logo">
+                <nav calss="menu">
+                    <a href="#" class="home"> <strong>Meet Karma</strong></a>
+                    <a href="#" class="home">How it Works</a>
+                    <a class="home" id="active" href="store.html" class="home">Store</a>
+                    <a href="#" class="home">Blog</a>
+                    <a href="#" class="home">Help</a>
+                    <a href="#" class="home">Login</a>
+                </nav>
+            </div>
+        </header>
+    </main>
+    <div class="body1">
+        <p class="p1">
+            <strong> Introducing Karma</strong>
+        </p>
+        <p class="p1">Bring Wifi with you, everywhere you go.</p>
+        <button class="button1">Learn More</button>
+    </div>
+    <br><br><br><br><br><br>
+    <div class="body2">
+
+        <div class="text3">
+            <h1>Everyone needs a little Karama</h1>
+        </div>
+
+    </div>
+    <div class="body3">
+
+        <div class="img3">
+            <img src="img/icon-devices.svg" class="devic">
+            <h4>Internet for all device</h4>
+        </div>
+
+        <div class="img3">
+            <img src="img/icon-coffee.svg" class="devic">
+            <h4>Boost your productivity</h4>
+        </div>
+        <div class="img3">
+            <img src="img/icon-refill.svg" class="devic">
+            <h4>Pay as You Go </h4>
+        </div>
+    </div>
+    <div class="contener">
+
+        <img class="img_2" src="level-2/homepage-feature.png">
 
+
+        <div class="text2">
+
+            <h1 class="contentquote"><span class="quote">&ldquo;</span>Wherever I am,I just don't <br>
+                worry about my connection<br> anymore!<span class="quote">&rdquo;</span></h1>
```

Comment:
You should not use <br> tag in within your paragraph, because the br tag makes new line, and you don't need a new line before the paragraph ends, instead make your container flexible so the text can resize and adjust accordingly.